thirdparty/nanosvg/.gitignore

```diff
@@ -0,0 +1,31 @@
+## Compiled source #
```

Comment:
You can also remove the README, we only need the LICENSE.txt details. Then add an entry for nanosvg in `thirdparty/README.md` to document the provenance of the lib.